src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,9 @@
-export const UserInfo = () => {};
+import { UserInfoProps } from '../../types';
+
+export const UserInfo = ({ user }: UserInfoProps) => {
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
```

Comment:
Checking for user could be moved to TodoInfo before rendering the component (as suggested in checklist 5)